azurerm/resource_arm_function_app.go

```diff
@@ -205,7 +205,6 @@ func resourceArmFunctionApp() *schema.Resource {
 						"virtual_network_name": {
 							Type:     schema.TypeString,
 							Optional: true,
-							Computed: false,
```

Comment:
is this intentional?

Reply:
Yea, `Computed: false,` doesn't actually mean anything right?